deep-security-agent/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Please refer to [GitHub Deep-Security-Agent cookbook releases](https://github.com/deep-security/chef/releases) for complete change log.
 
+## [1.1.2 - 2018-04-18]
+
+## CHANGED
```

Comment:
This should be `FIXED`, because it is a bug fix.
based on https://ruby-doc.org/core-2.4.0/String.html#method-i-casecmp